docs/shared/migration/angular-integrated.md

```diff
@@ -1,14 +1,14 @@
 # Migrating an Angular CLI workspace to an Integrated Nx Monorepo
 
-To take advantage of Nx's monorepo features provided by Nx and the Nx Angular plugin, you can also perform a migration from an Angular CLI to an Integrated Nx Monorepo with the command:
+To take advantage of Nx's monorepo features provided by Nx and the Nx Angular plugin, you can also perform a migration from an Angular CLI workspace to an [Integrated Nx Monorepo](/concepts/integrated-vs-package-based#integrated-repos) with the command:
```

Comment:
Let's make it more concise maybe

"If you want to migrate your Angular CLI project to an [Integrated Nx Monorepo](/concepts/integrated-vs-package-based#integrated-repos), use the following command:"